gui/src-tauri/src/main.rs

```diff
@@ -64,6 +63,14 @@ fn get_launch_path(cli: Cli) -> Option<PathBuf> {
 		}
 	}
 
+	// This is only for AppImage
+	if let Some(appimage) = env::var_os("APPDIR") {
+		let path = PathBuf::from(appimage);
+		if is_valid_path(&path) {
```

Comment:
you call the same code 4 times. Why not just construct a list from each possible path source, loop on it, and return on first `Some`.

Reply:
Not related to the current PR

Reply:
It is because you added another thing to this section of code. Its a simple fix, should be easy to do in under 3 minutes.

Reply:
oh I understand now - the appimage path stuff was already added in a prior PR. In that case, resolving.